### ISSUE TITLE
Bug displaying applications with trackers.

### ISSUE THREAD

**Issue by tommynok** [MuntashirAkon/AppManager#506]
Total number of applications installed on the phone is 345.
But when I try to batch disable trackers through "1-click ops" - "block/unblock trackers"  I see only 5 applications.

Although when looking at the components of other applications, I see that there are trackers in a fairly large number of installed and pre-installed applications.

AM version - 2.7.0-1388 (390)
Device - Lenovo P1C58
OS version - 6.0.1
Mode - root
Root - SuperSU (sysremless)
[logs-2021-07-07-15-55-47.zip](https://github.com/MuntashirAkon/AppManager/files/6777382/logs-2021-07-07-15-55-47.zip)
[2021-07-07_155736_logcat.zip](https://github.com/MuntashirAkon/AppManager/files/6777383/2021-07-07_155736_logcat.zip)

**Comment by MuntashirAkon** [comment]
I can see nothing because the logs are filtered using the package name.

**Comment by MuntashirAkon** [comment]
Found the issue:

```
E/JavaBinder( 6472): !!! FAILED BINDER TRANSACTION !!!  (parcel size = 4)
W/ParceledListSlice( 6472): Failure retrieving array; only received 6 of 250
W/ParceledListSlice( 6472): android.os.DeadObjectException: Transaction failed on small parcel; remote process probably died
W/ParceledListSlice( 6472): 	at android.os.BinderProxy.transactNative(Native Method)
W/ParceledListSlice( 6472): 	at android.os.BinderProxy.transact(Binder.java:503)
W/ParceledListSlice( 6472): 	at android.content.pm.ParceledListSlice.<init>(ParceledListSlice.java:97)
W/ParceledListSlice( 6472): 	at android.content.pm.ParceledListSlice.<init>(ParceledListSlice.java:41)
W/ParceledListSlice( 6472): 	at android.content.pm.ParceledListSlice$2.createFromParcel(ParceledListSlice.java:200)
W/ParceledListSlice( 6472): 	at android.content.pm.ParceledListSlice$2.createFromParcel(ParceledListSlice.java:198)
W/ParceledListSlice( 6472): 	at android.content.pm.IPackageManager$Stub$Proxy.getInstalledPackages(IPackageManager.java:3323)
W/ParceledListSlice( 6472): 	at android.app.ApplicationPackageManager.getInstalledPackages(ApplicationPackageManager.java:595)
W/ParceledListSlice( 6472): 	at android.app.ApplicationPackageManager.getInstalledPackages(ApplicationPackageManager.java:588)
W/ParceledListSlice( 6472): 	at io.github.muntashirakon.AppManager.oneclickops.OneClickOpsActivity.lambda$blockTrackers$14$OneClickOpsActivity(OneClickOpsActivity.java:141)
W/ParceledListSlice( 6472): 	at io.github.muntashirakon.AppManager.oneclickops.-$$Lambda$OneClickOpsActivity$9bYTndTc1uuxZhit_EXf37i69BI.run(lambda)
W/ParceledListSlice( 6472): 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:423)
W/ParceledListSlice( 6472): 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
W/ParceledListSlice( 6472): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
W/ParceledListSlice( 6472): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
W/ParceledListSlice( 6472): 	at java.lang.Thread.run(Thread.java:818)
```

and

```
E/JavaBinder( 6571): !!! FAILED BINDER TRANSACTION !!!  (parcel size = 140)
E/JavaBinder( 6571): *** Uncaught remote exception!  (Exceptions are not yet supported across processes.)
E/JavaBinder( 6571): java.lang.RuntimeException: android.os.DeadObjectException: Transaction failed on small parcel; remote process probably died
E/JavaBinder( 6571): 	at android.os.Parcel.writeException(Parcel.java:1520)
E/JavaBinder( 6571): 	at android.os.Binder.execTransact(Binder.java:459)
E/JavaBinder( 6571): Caused by: android.os.DeadObjectException: Transaction failed on small parcel; remote process probably died
E/JavaBinder( 6571): 	at android.os.BinderProxy.transactNative(Native Method)
E/JavaBinder( 6571): 	at android.os.BinderProxy.transact(Binder.java:503)
E/JavaBinder( 6571): 	at io.github.muntashirakon.AppManager.ipc.AMService$IAMServiceImpl.transactRemote(AMService.java:144)
E/JavaBinder( 6571): 	at io.github.muntashirakon.AppManager.ipc.AMService$IAMServiceImpl.onTransact(AMService.java:118)
E/JavaBinder( 6571): 	at android.os.Binder.execTransact(Binder.java:453)
```

This issue is similar to #298.

**Comment by MuntashirAkon** [comment]
Fixed in f73796c4
